benefits/enrollment/templates/enrollment/index.html

```diff
@@ -12,23 +12,21 @@
 {% endblock nav-buttons %}
 
 {% block headline %}
-  <div class="col-lg-8">
-    <h1 class="pb-lg-8 pb-4">
-      {% translate "Your eligibility is confirmed!<br>You’re almost there." %}
-    </h1>
+  <div class="col-lg-6">
+    <h1 class="pb-4">{% translate "Your eligibility is confirmed! You’re almost there." %}</h1>
   </div>
 {% endblock headline %}
 
 {% block inner-content %}
-  {% block media-items %}
-    <div class="col-12 col-sm-12 col-lg-8">
-      <ul class="d-flex list-unstyled ps-0 pb-5 mb-3">
+  <div class="col-12 col-lg-6">
+    {% block media-items %}
+      <ul class="d-flex list-unstyled ps-0 mb-0">
         {% include "enrollment/includes/media-item--contactless-card--index.html" %}
       </ul>
-    </div>
-  {% endblock media-items %}
-  {% block additional-info %}
-  {% endblock additional-info %}
+    {% endblock media-items %}
+    {% block additional-info %}
+    {% endblock additional-info %}
+  </div>
```

Comment:
This eliminates some [`div`itis](https://www.wordnik.com/words/divitis):

<img width="965" alt="image" src="https://github.com/user-attachments/assets/4b808be2-58e0-49f0-930d-9333ea174bfd">

Instead of having three `<div class="row justify-content-center"><div class="col-lg-6"></div></div>`, we just have 1.